deploy-manage/users-roles/cluster-or-deployment-auth/user-roles.md

```diff
@@ -1,10 +1,39 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/elasticsearch/reference/current/authorization.html
+applies:
+  hosted: all
+  ece: all
+  eck: all
+  stack: all
 ---
 
 # User roles [authorization]
 
+
+After a user is authenticated, use role-based access control to determine whether the user behind an incoming request is allowed to execute the request. The primary method of authorization in a cluster is role-based access control (RBAC). Review the following topics to learn about authorization in your Elasticsearch cluster.
+
+### Set up user authorization
+
+* [Define roles](/deploy-manage/users-roles/cluster-or-deployment-auth/defining-roles.md)
+* Learn about [built-in roles](/deploy-manage/users-roles/cluster-or-deployment-auth/built-in-roles.md)
+* Learn about the [Elasticsearch](/deploy-manage/users-roles/cluster-or-deployment-auth/elasticsearch-privileges.md) and [Kibana](/deploy-manage/users-roles/cluster-or-deployment-auth/kibana-privileges.md) privileges you can assign to roles
+* [Map users and groups to roles](/deploy-manage/users-roles/cluster-or-deployment-auth/mapping-users-groups-to-roles.md)
```

Comment:
I would specify that this is only used with external realms, as it's not the assignment of a normal user (native) to roles.

You could call it also `Map external users and groups to roles` if you want to make it clearer.

What I want to avoid is the confusion about that being a generic mapping of user to role, because it's not, it's only for users that don't exist in elasticsearch because they are LDAP, SAML, OIDC, etc...